.github/workflows/tests-smoke-ipv6.yaml

```diff
@@ -103,58 +103,70 @@ jobs:
         shell: bash
         run: |
           for image in cilium-ci operator-generic-ci hubble-relay-ci ; do
-            until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.vars.outputs.tag }} &> /dev/null; do sleep 45s; done
+            until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.sha.outputs.tag }} &> /dev/null; do sleep 45s; done
           done
 
-      - name: Install cilium chart
+      - name: Set up install variables
+        id: vars
+        run: |
+          CILIUM_INSTALL_DEFAULTS="--chart-directory=install/kubernetes/cilium \
+            --helm-set nodeinit.enabled=true \
+            --helm-set kubeProxyReplacement=strict \
+            --helm-set ipam.mode=kubernetes \
+            --helm-set image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci \
+            --helm-set image.tag=${{ steps.sha.outputs.tag }} \
+            --helm-set image.pullPolicy=IfNotPresent \
+            --helm-set image.useDigest=false \
+            --helm-set hubble.enabled=true \
+            --helm-set hubble.relay.enabled=true \
+            --helm-set hubble.relay.image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci \
+            --helm-set hubble.relay.image.tag=${{ steps.sha.outputs.tag }} \
+            --helm-set hubble.relay.image.pullPolicy=IfNotPresent \
+            --helm-set hubble.relay.image.useDigest=false \
+            --helm-set operator.image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/operator \
+            --helm-set operator.image.suffix=-ci \
+            --helm-set operator.image.tag=${{ steps.sha.outputs.tag }} \
+            --helm-set operator.image.pullPolicy=IfNotPresent \
+            --helm-set operator.image.useDigest=false \
+            --helm-set ipv6.enabled=true \
+            --helm-set ipv4.enabled=false \
+            --helm-set routingMode=native \
+            --helm-set autoDirectNodeRoutes=true \
+            --helm-set ipv6NativeRoutingCIDR=2001:db8:1::/64 \
+            --helm-set ingressController.enabled=true \
+            --helm-set bpf.monitorAggregation=none \
+            --version="
+
+          echo cilium_install_defaults=${CILIUM_INSTALL_DEFAULTS} >> $GITHUB_OUTPUT
+
+      - name: Install Cilium CLI
+        run: |
+          curl -sSL --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${{ env.cilium_cli_version }}/cilium-linux-amd64.tar.gz{,.sha256sum}
+          sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
+          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
+          rm cilium-linux-amd64.tar.gz{,.sha256sum}
+          cilium version
```

Comment:
Hi @bleggett , was there a reason that this PR removes the usage of the CLI action and switches to directly encoding the instructions here? Is it possible this was introduced due to incorrect rebase against https://github.com/cilium/cilium/pull/25228 ?

See below the newer install approach:
```
      - name: Install Cilium CLI
￼        if: ${{ failure() }}
￼        uses: cilium/cilium-cli@d4c49cddefadf11852cd5bbde0bbd9e0b2c67d43 # v0.14.3
￼        with:
￼          release-version: ${{ env.cilium_cli_version }}
￼          ci-version: ${{ env.cilium_cli_ci_version }}
```

Reply:
No, no reason, IIRC there were 2-3 different ways jobs in this file did it when I merged the PR.

Action is def. better and I'll fix it unless there's already a pending fix.

Reply:
@joestringer https://github.com/cilium/cilium/pull/28661 and apologies, not sure why I didn't just use the action.

Reply:
no worries, thanks for the quick fix.